platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegionError.java

```diff
@@ -0,0 +1,47 @@
+package com.mapbox.mapboxsdk.offline;
+
+/**
+ * Created by antonio on 2/21/16.
+ */
+public class OfflineRegionError {
+
+    /**
+     * Success = 1
```

Comment:
The idiomatic version in Android is to use the `@IntDef` annotation like I had used for `OfflineRegion.DownloadState` (we try to avoid enums at the SDK level). Changes are in https://github.com/mapbox/mapbox-gl-native/commit/9eb74359f07da2f13807a807f043f55b47f68137.